lib/league_statable.rb

```diff
@@ -70,16 +70,14 @@ def lowest_scoring_home_team
  
 
   # Method to determine count of unique teams in a CSV file
-    def count_of_teams(game_teams_path)
+# Method to determine count of unique teams in a CSV file
+    def count_of_teams
         team_ids = []
-       
 
-    CSV.foreach(game_teams_path, headers: true) do |row|
```

Comment:
Why are you call csv rather than accessing the data directly from instance variables?

Reply:
same deal as below, i dont have these, those were the deletions

Reply:
Cool!  I'm apparently color blinded